generated/nidigital/nidigital/enums.py

```diff
@@ -6,69 +6,165 @@
+    r'''
+    Pattern Label History RAM trigger
+    '''
 
 
 class PPMUApertureTimeUnits(Enum):
     SECONDS = 2100
+    r'''
+    Unit in seconds.
+    '''
 
 
 class PPMUCurrentLimitBehavior(Enum):
     REGULATE = 3100
+    r'''
+    Controls output current so that it does not exceed the current limit. Power continues to generate even if the current limit is reached.
+    '''
 
 
 class PPMUMeasurementType(Enum):
     CURRENT = 2400
+    r'''
+    The PPMU measures current.
+    '''
     VOLTAGE = 2401
+    r'''
+    The PPMU measures voltage.
+    '''
 
 
 class PPMUOutputFunction(Enum):
     VOLTAGE = 1300
+    r'''
+    The PPMU forces voltage to the DUT.
+    '''
     CURRENT = 1301
+    r'''
+    The PPMU forces current to the DUT.
+    '''
 
 
 class PinState(Enum):
     ZERO = 0
+    r'''
+    A digital state of 0. This state can be used for WriteStatic(PinState).
+    '''
     ONE = 1
+    r'''
+    A digital state of 1. This state can be used for WriteStatic(PinState).
+    '''
     L = 3
+    r'''
+    A digital state of L (low). This state should not be used with WriteStatic(PinState).
+    '''
     H = 4
+    r'''
+    A digital state of H (high). This state should not be used with WriteStatic(PinState).
+    '''
     X = 5
+    r'''
+    A digital state of X (non-drive state). This state can be used for WriteStatic(PinState).
+    '''
     M = 6
+    r'''
+    A digital state of M (midband). This state should not be used with WriteStatic(PinState).
+    '''
     V = 7
+    r'''
+    A digital state of V (compare high or low, not midband; store results from capture functionality if configured). This state should not be used with WriteStatic(PinState).
+    '''
     D = 8
+    r'''
+    A digital state of D (drive data from source functionality if configured). This state should not be used with WriteStatic(PinState).
+    '''
     E = 9
     NOT_A_PIN_STATE = 254
+    r'''
+    Not a pin state is used for non-existent DUT cycles.
+    '''
     PIN_STATE_NOT_ACQUIRED = 255
 
 
 class SelectedFunction(Enum):
     DIGITAL = 1100
+    r'''
+    The pattern sequencer controls the specified pin(s). If a pattern is currently bursting, the pin immediately switches to bursting the pattern. This option disconnects the PPMU.
+    '''
     PPMU = 1101
+    r'''
+    The PPMU controls the specified pin(s) and connects the PPMU. The pin driver is in a non-drive state, and the active load is disabled. The PPMU does not start sourcing or measuring until Source or Measure(PpmuMeasurementType) is called.
+    '''
     OFF = 1102
+    r'''
+    Puts the digital driver in a non-drive state, disables the active load, disconnects the PPMU, and closes the I/O switch connecting the instrument channel.
+    '''
     DISCONNECT = 1103
+    r'''
+    The I/O switch connecting the instrument channel is open to the I/O connector. If the PPMU is sourcing, it is stopped prior to opening the I/O switch.
+    '''
 
 
 class SequencerFlag(Enum):
```

Comment:
I guess there's not much value in adding docstrings for the enum values. Maybe a docstring for the enum class itself?

(Applies to `SequencerRegister` too)

Reply:
I tried adding a ['documentation']['description'] to the metadata for the enum in src/nidigital/metadata/enum.py, but it was ignored.  I also can't find any examples of other drivers in nimi-python documenting the enum class rather than the individual values.

Reply:
I guess code generator doesn't support it.